src/alt.js

```diff
@@ -460,7 +460,7 @@ class Alt {
   takeSnapshot(...storeNames) {
     const state = snapshot(this, ...storeNames)
     if (this[LAST_SNAPSHOT]) {
-      assign(this[LAST_SNAPSHOT], state)
+      this[LAST_SNAPSHOT] = JSON.stringify(assign(JSON.parse(this[LAST_SNAPSHOT]), JSON.parse(state)))
```

Comment:
Wondering if there is a reason we set `LAST_SNAPSHOT` to null initially, rather than just making it an empty object like we do with `INIT_SNAPSHOT`.  Doing so could clean this up even more.